### PR TITLE
feat: add load test

### LINKED ISSUE #298: Load test: many requests
This is related to https://github.com/GenSpectrum/LAPIS/issues/666

Try to find out if SILO has problems with many requests.

### PR DESCRIPTION
**Pull request by JonasKellerer**
resolves #298

### Summary
This introduces a new python script which runs the k6 load test. Running `python3 loadtest.py --help` will show the program arguments.

## PR Checklist
- [ ] All necessary documentation has been adapted or there is an issue to do so.
- [ ] The implemented feature is covered by an appropriate test.

### CODE REVIEW

**Comment by JonasKellerer** [comment]
And the documentation is missing here. But maybe we discuss first if this is helpful :)

File: loadTests/loadtest.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        try:
+            return TestActions[arg.upper()]
+        except KeyError:
+            raise ValueError(f"Invalid action: {arg}. Valid actions are: AGGREGATED, MUTATIONS")
+
+    def filter_type(arg: str) -> TestFilters:
+        try:
+            return TestFilters[arg.upper()]
+        except KeyError:
+            raise ValueError(f"Invalid filter: {arg}. Valid filters are: TRUE, INT_EQUALS, COMPLEX_MUTATION")
```
Comment (Taepper) [inline loadTests/loadtest.py:56]: Can you add the valid filters to the help message of this option?

File: loadTests/loadtest.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    parser.add_argument('--action', type=str, default='AGGREGATED', help='Query to send to SILO')
+    parser.add_argument('--filter', type=str, default='TRUE', help='Filter for query')
+
+    args = parser.parse_args()
+
+    def action_type(arg: str) -> TestActions:
+        try:
+            return TestActions[arg.upper()]
+        except KeyError:
+            raise ValueError(f"Invalid action: {arg}. Valid actions are: AGGREGATED, MUTATIONS")
```
Comment (Taepper) [inline loadTests/loadtest.py:50]: see below

**Comment by JonasKellerer** [comment]
FYI: The docker command uses `--network='host'`. This is used for linux, to get access to localhost on that system. For mac and windows this might be different.

**Comment by JonasKellerer** [comment]
Not needed any longer. We will perform other load tests when necessary.

**Pull request closed without merging**